src/modules/previewpane/common/packages.config

```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Updated the Common project to use `Package References` instead of `packages.config`. More [details here.](https://docs.microsoft.com/en-us/nuget/consume-packages/migrate-packages-config-to-package-reference#benefits-of-using-packagereference)